src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/FhirJsonInputFormatter.cs

```diff
@@ -95,7 +95,8 @@ public override Task<InputFormatterResult> ReadRequestBodyAsync(InputFormatterCo
 
                 if (delayedException != null)
                 {
-                    var errorMessage = string.IsNullOrEmpty(delayedException.Message) ? Api.Resources.ParsingError : delayedException.Message;
+                    // Avoid using the exception's message, since the exception thrown in the HL7 FHIR core library has an incorrect message.
```

Comment:
Is there a place where we can log bugs that originate in FHIR libraries?

Reply:
We can create issues in that repo: https://github.com/FirelyTeam/fhir-net-api.
If it blocks us, we have also submitted fixes in the past.

Reply:
Awesome thanks, I created [#1304](https://github.com/FirelyTeam/fhir-net-api/issues/1304).